js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeRequestedPartitionsLink.tsx

```diff
@@ -63,15 +66,22 @@ export const AutomaterializeRequestedPartitionsLink = ({runIds, partitionKeys}:
     }
 
     return runIds ? (
-      <PartitionAndRunList runIds={runIds} partitionKeys={filteredPartitionKeys} />
+      <PartitionAndRunList runIds={runIds} partitionKeys={filteredPartitionKeys} intent={intent} />
     ) : (
       <VirtualizedPartitionList partitionKeys={partitionKeys} />
     );
   };
 
   return (
     <>
-      <ButtonLink onClick={() => setIsOpen(true)}>{label}</ButtonLink>
+      <Box>
+        <Tag intent={intent}>{label}</Tag>
+        <ButtonLink onClick={() => setIsOpen(true)}>
+          <small style={{marginLeft: 10}}>
+            <a>View details</a>
```

Comment:
This doesn't need to be an `a`.

To be more precise, it *shouldn't* be an `a`. The rendered component for `ButtonLink` is a `button` element, which should never contain other interactive elements like `a`. (Similarly, an `a` should never contain a `button`, etc.)